purchase_request_tier_validation/data/purchase_request_tier_definition.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2019 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
```

Comment:
`<odoo noupdate="1">`, No need to Data

Reply:
Done :+1: Can you update review, please?